tests/test_jobresource.py

```diff
@@ -16,6 +18,7 @@ def _test_prints_objects(self, cmd_mod, resource_name):
         jobid = '1/2/3'
         with mock.patch.object(cmd_mod, 'get_job', autospec=True) as mock_gj:
             # Patch job.items.iter_json() to return our objects
+            mock_gj.return_value._metadata_updated = time.time()
```

Comment:
It worked before because Py2 allowed more slack in comparison operators, in Py3 the following failed:
```
File "/home/immerrr/scrapinghub/shub/shub/utils.py", line 454, in job_live 
    if time.time() - job._metadata_updated > refresh_meta_after:
TypeError: unorderable types: MagicMock() > int()
```